eqt/ui/UIFormWidget.py

```diff
@@ -337,7 +337,7 @@ def __init__(self, parent=None):
 
 class FormDockWidget(QtWidgets.QDockWidget):
     def __init__(self, parent=None, title=None):
-        QtWidgets.QDockWidget.__init__(self, parent)
+        QtWidgets.QDockWidget.__init__(self, title, parent)
```

Comment:
You may need a check of whether title is None or not before passing it here?

Reply:
Or perhaps title=tile, parent=parent may be sufficient?

Reply:
I tried 

 `QtWidgets.QDockWidget.__init__(self, title=title, parent=parent)`

and it did not work.

Reply:
I did the following

`if title == None:
            title = ''`

because it seems that `title` accepts str type but not None type. If you find a better solution please let me know, I could not find anything better.